docker/openchallenges/services/mcp-server-standalone.yml

```diff
@@ -0,0 +1,14 @@
+services:
+  openchallenges-mcp-server:
+    image: ghcr.io/sage-bionetworks/openchallenges-mcp-server:latest
```

Comment:
@tschaffter  I am not familiar with the process to update the image in GHCR, but the `latest` version is not included the env variables we recently added (e.g. `OPENCHALLENGES_MCP_SERVER_WELCOME_MESSAGE)`. Should we keep using the [`latest` tag](https://github.com/orgs/Sage-Bionetworks/packages/container/openchallenges-mcp-server/416657194?tag=latest) or [`edge` tag](https://github.com/orgs/Sage-Bionetworks/packages/container/openchallenges-mcp-server/419018125?tag=edge)

Reply:
See comment below.